Container-mod examples (#73)
ubccr/ccr-examples: containers/1_Advanced/container-mod/README.md

- `-j, --jupyter`: Creates a Jupyter kernel for the containerized software, if compatible
- `-h, --help`: For built-in help

For detailed information about the topics above and additional options, please refer to the official [`container-mod` documentation](https://github.com/TuftsRT/container-mod#usage).

2. Follow the example for the workflow that aligns with your use case:  
- [`personal/`](https://github.com/ubccr/ccr-examples/tree/main/containers/1_Advanced/container-mod/personal) - `--personal` mode
- [`project/`](https://github.com/ubccr/ccr-examples/tree/main/containers/1_Advanced/container-mod/project) - `--profile` mode

3. Add your module path to your environment
In order to see the modules you generate with `container-mod` you need to point your account to the path where the modules are stored.  You can do this each time you want to run these modules by first running this for `--personal` mode:	
```
module use ~/privatemodules
```
OR this for `--profile` mode:
```
module use /projects/academic/[YourGroupName]/privatemodules
```
However, this is forgotten on logout and you'd need to specify this in your Slurm batch scripts whenever you run jobs.

> To update the $PATH environment persistently and for each login & job submission, edit the `~/.ccr/modulepaths` file to include this for `--personal` mode:

Reviewer: ```suggestion
To update the $PATH environment persistently and for each login & job submission, edit the `~/.ccr/modulepaths` file

 For `--personal` mode add the following:
```

`/user/[CCRUsername]/privatemodules`

> OR this for `--profile` mode:

Reviewer: ```suggestion
For `--profile` mode add the following:
```

```
/projects/academic/[YourGroupName]/privatemodules
```
**With either option, make sure to LOGOUT and back in again to see the change.**

4. Once done, verify `Lmod` recognizes the new module with commands like `echo $MODULEPATH`, `module avail`, `module show`, etc.

> Refer to the official `container-mod` [documentation](https://github.com/TuftsRT/container-mod) for more information.

Reviewer: ```suggestion
Refer to the official [container-mod documentation](https://github.com/TuftsRT/container-mod/docs) for more information.
```

Author: Since this link doesn't work, I used the new container-mod readthedocs link.  The developer says it's a draft but it's better than looking at the docs in github
